apps/apple/package.json

```diff
@@ -21,10 +21,12 @@
     "babel-jest": "^24.9.0",
```

Comment:
Github won't let me comment on the top of the file, but it might be worth changing the "name" of the package to `FluentUITester-apple` to match the other platform's packages for their test app

Reply:
Aren't we about to rename it to be ios/macos specific next though?